src/cargo/ops/cargo_package/mod.rs

```diff
@@ -40,10 +41,38 @@ use unicase::Ascii as UncasedAscii;
 mod vcs;
 mod verify;
 
+/// Message format for `cargo package`.
+///
+/// Currently only affect the output of the `--list` flag.
+#[derive(Debug, Clone)]
+pub enum PackageMessageFormat {
+    Plain,
+    Json,
+}
+
+impl PackageMessageFormat {
+    pub const POSSIBLE_VALUES: [&str; 2] = ["plain", "json"];
+
+    pub const DEFAULT: &str = "plain";
```

Comment:
Should `plain` be `human`?  Thats what we use for `cargo check`

Reply:
Changed it to `human`. If we're going to `--message-format` for every message, the default will be `human`. The current file-per-line format should be "human" since it is the default.